src/main/java/org/springframework/data/couchbase/core/support/WithProjecting.java

```diff
@@ -0,0 +1,27 @@
+/*
```

Comment:
is that interface actually used somwhere in this pr?

Reply:
It's not.  It was intended to be used on all classes that have this project() api, but I couldn't figure out how to resolve this project() method with the one being implemented.